data/mods/MindOverMatter/effectoncondition/eoc_nether_attunement_events.json

```
@@ -522,6 +524,36 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_HALLUCINATIONS",
+    "condition": {
+      "and": [
+        { "math": [ "u_vitamin('vitamin_psionic_drain')", ">=", "125" ] },
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "clamp( (u_vitamin('vitamin_psionic_drain') - 100), 0, 50) + (nether_attune_difficulty_scaler(u_latest_channeled_power_difficulty)) + 10"
```

```suggestion
               "( clamp( ( u_vitamin('vitamin_psionic_drain')  - 160), 0, 100) + clamp( ( (u_vitamin('vitamin_psionic_drain') - 210) * 3.5 ), 0, 150) + (nether_attune_difficulty_scaler(u_latest_channeled_power_difficulty)) + 10)"          
```
 Just changing this to the "first slow, then fast" scale the others have, to match how much bonus your powers get.